app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,57 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: %i[show edit update destroy]
+  before_action :set_project
+  def index
+    @tasks = Task.where(project: @project)
```

Comment:
Since you have a `has_many` association, you can write like this:
```suggestion
    @tasks = @project.tasks
```